BREAKING.md

```diff
@@ -112,6 +113,15 @@ The `@fluidframework/web-code-loader` and the `ICodeAllowList` were deprecated i
 +   }
 ```
 
+### Remove deprecated PendingStateManager interfaces
+
+The following interfaces used by the `PendingStateManager` are no longer be exported:
```

Comment:
```suggestion
The following interfaces used by the `PendingStateManager` are no longer exported:
```

Reply:
I also assume there's no workaround or alternatives for these, right?

Reply:
No workaround or alternatives. They shouldn't be used anywhere and it's a case of exporting too much.